src/big_query.mli

```diff
@@ -29,6 +29,20 @@ end
 module Datasets : sig
   val get : ?project_id:string -> dataset_id:string -> unit -> (string, [> Error.t ]) Lwt_result.t
   val list : ?project_id:string -> unit -> (string, [> Error.t ]) Lwt_result.t
+
+  module Tables : sig
+
+    type table =
+      { id : string;
+        creationTime : string }
+
+    type resp =
+      { tables : table list;
+        totalItems : int }
```

Comment:
Does the response have a `nextPage` field? We'll need it to use `page_token`

Reply:
it's `nextPageToken`